### PR TITLE
chore(deps): upgrade swagger-ui-react to 4.11.1

### PR DESCRIPTION
**Pull request by js0cha**
Mitigate the current XSS vulnerabilities (https://www.npmjs.com/advisories/1070339)

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/tender-weeks-help.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-api-docs': major
```
Comment (benjdlambert) [inline .changeset/tender-weeks-help.md:2]: This should be a patch release for this :pray:

**Review by benjdlambert — COMMENTED** [review]
Thanks for this, also needs a DCO in the commit. Check the bot automated message for help 🙇

File: .changeset/tender-weeks-help.md (hunk 1 above)
Comment (js0cha) [reply .changeset/tender-weeks-help.md]: Okay

**Review by jhaals — COMMENTED** [review]
Thank you!

Would be nice if you could tweak the changeset slightly, perhaps change it to

```
Updated `swagger-ui-react` to 4.11.1 in order to address a [XSS vulnerability](https://github.com/advisories/GHSA-hqq7-2q2v-82xq) in `@braintree/sanitize-url`
```

**Comment by freben** [comment]
Could you run prettier on the file as well? See the build error.

**Comment by freben** [comment]
Alright I made a fixup on top, let's see if it goes green

**Review by freben — APPROVED** [review]

**Pull request merged**